tests/Writer/XlsxWriterTest.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\Exporter\Tests\Writer;
+
+use PhpOffice\PhpSpreadsheet\IOFactory;
+use Sonata\Exporter\Test\AbstractTypedWriterTestCase;
+use Sonata\Exporter\Writer\TypedWriterInterface;
+use Sonata\Exporter\Writer\XlsxWriter;
+
+final class XlsxWriterTest extends AbstractTypedWriterTestCase
+{
+    protected $filename;
```

Comment:
If this class is final, why these properties need to be protected?

Reply:
Certainly copied pasted from another test but indeed it can be private

Reply:
Yes, copied from the other tests. Made them private now.